2025-08-08,13:55:35 [INFO] (app.py:app:316): Parsing config file /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server/app/../common/default.conf
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter port set to "9004" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter server_version set to "2.13.0" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter evaluate_enabled set to "True" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter evaluate_timeout set to "30" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter upload_dir set to "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tmp/query_objects" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter transfer_protocol set to "http" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:284): Parameter certificate_file is not set
2025-08-08,13:55:35 [DEBUG] (app.py:app:284): Parameter key_file is not set
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter minimum_tls_version set to "TLSv1_2" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter state_file_path set to "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter static_path set to "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server/static" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:284): Parameter TABPY_PWD_FILE is not set
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter log_request_context set to "false" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter max_request_size_in_mb set to "100" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter gzip_enabled set to "True" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter arrow_enabled set to "False" from default value
2025-08-08,13:55:35 [DEBUG] (app.py:app:277): Parameter arrowflight_port set to "13622" from default value
2025-08-08,13:55:35 [INFO] (app.py:app:381): Setting max request size to 104857600 bytes
2025-08-08,13:55:35 [DEBUG] (app.py:app:539): File /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server/state.ini not found, creating from template /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server/state.ini.template...
2025-08-08,13:55:35 [INFO] (app.py:app:545): Loading state from state file /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server/state.ini
2025-08-08,13:55:35 [DEBUG] (app.py:app:397): Static pages folder set to "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/tabpy/tabpy_server/static"
2025-08-08,13:55:37 [INFO] (app.py:app:510): Password file is not specified: Authentication is not enabled
2025-08-08,13:55:37 [INFO] (app.py:app:429): Call context logging is disabled
2025-08-08,13:55:37 [INFO] (app.py:app:197): Initializing TabPy...
2025-08-08,13:55:37 [DEBUG] (selector_events.py:selector_events:64): Using selector: KqueueSelector
2025-08-08,13:55:37 [INFO] (callbacks.py:callbacks:43): Initializing TabPy Server...
2025-08-08,13:55:37 [DEBUG] (state.py:state:628): Loading option 'None' from section [Query Objects Service Versions]...
2025-08-08,13:55:37 [DEBUG] (state.py:state:650): Returning value '[]'
2025-08-08,13:55:37 [DEBUG] (state.py:state:148): Collected endpoints: {}
2025-08-08,13:55:37 [INFO] (app.py:app:201): Done initializing TabPy.
2025-08-08,13:55:37 [INFO] (callbacks.py:callbacks:64): Initializing models...
2025-08-08,13:55:37 [DEBUG] (state.py:state:628): Loading option 'None' from section [Query Objects Service Versions]...
2025-08-08,13:55:37 [DEBUG] (state.py:state:650): Returning value '[]'
2025-08-08,13:55:37 [DEBUG] (state.py:state:148): Collected endpoints: {}
2025-08-08,13:55:37 [INFO] (app.py:app:167): Web service listening on port 9004
